unit_tests/sources/declarative/test_manifest_declarative_source.py

```diff
@@ -627,7 +627,6 @@ def test_source_missing_checker_fails_validation(self):
                     },
                 }
             ],
-            "check": {"type": "CheckStream"},
```

Comment:
I was getting test failures here and my suspicion is that because we no longer made `stream_names` a required field since it also supports dynamic streams check configs. This was in https://github.com/airbytehq/airbyte-python-cdk/pull/450

So then this component is actually valid and does not throw the Validation error anymore. Getting rid of this component entirely causes the test to correctly fail.

What I don't understand is why the tests didn't fail on the above PR. Nonetheless getting rid of this should fix my build

Reply:
What were the test failures? Does that means that https://github.com/airbytehq/airbyte-python-cdk/pull/450 was a breaking change?

Reply:
basically this test was no longer raising a `ValidationError` despite expecting it. 
```
Failed: DID NOT RAISE <class 'jsonschema.exceptions.ValidationError'>
```

I wouldn't consider it to be a breaking change because we were relaxing the restrictions on what throws an error. So no existing manifests should break. Only tests that were asserting that the manifest validations raise errors like this test does. Make sense?

Reply:
Yes, that makes sense, thanks for the precision!